fetch/api/basic/request-headers.js

```diff
@@ -24,11 +42,15 @@ var url = RESOURCES_DIR + "inspect-headers.py"
 
 requestHeaders("Fetch with GET", url, "GET", null, location.origin, null);
 requestHeaders("Fetch with HEAD", url, "HEAD", null, location.origin, null);
-requestHeaders("Fetch with HEAD with body", url, "HEAD", "Request's body", location.origin, "14");
 requestHeaders("Fetch with PUT without body", url, "POST", null, location.origin, "0");
 requestHeaders("Fetch with PUT with body", url, "PUT", "Request's body", location.origin, "14");
 requestHeaders("Fetch with POST without body", url, "POST", null, location.origin, "0");
-requestHeaders("Fetch with POST with body", url, "POST", "Request's body", location.origin, "14");
+requestHeaders("Fetch with POST with text body", url, "POST", "Request's body", location.origin, "14");
+if (self.FormData)
```

Comment:
Rather than have a special case for browsers that don't support FormData, we should encapsulate the `new FormData()` argument inside of a function, catch any exception that is thrown and fail the test.